semantic_segmentation/scripts/dataset.py

```diff
@@ -29,13 +29,12 @@ def __getitem__(self, idx):
                                    self.targets[idx])
         img = Image.open(img_path).convert("RGB")
         img = np.array(img)
-        # img = img.resize((190, 275))
         target = Image.open(target_path).convert("L")
         target = np.array(target)
-        # target = target.resize((190, 275))
         if self.transforms is not None:
-            img, target = self.transforms(img, target)
-
+            transformed = self.transforms(image=img, mask=target)
+            img = transformed['image']
+            target = transformed['mask']
```

Comment:
good!

sometimes you want to just iterate ds without the transform - here you can do it without problem